cartographer_rviz/cartographer_rviz/submaps_display.cc

```diff
@@ -138,8 +138,8 @@ void SubmapsDisplay::processMessage(
       trajectory.emplace(
           submap_index,
           ::cartographer::common::make_unique<DrawableSubmap>(
-              trajectory_id, submap_index, context_->getSceneManager(),
-              trajectory_category.get(), visibility_all_enabled_->getBool()));
+              trajectory_id, submap_index, context_, trajectory_category.get(),
+              visibility_all_enabled_->getBool(), 0.3f, 0.06f));
```

Comment:
will we ever want to change these? If so, they should be configurable, if not they should be constexpr next to where they are used. I put my vote in for the second option for now.